sections/futures/Trade/MarketsDropdown.tsx

```diff
@@ -19,8 +19,9 @@ import useSelectedPriceCurrency from 'hooks/useSelectedPriceCurrency';
 import { formatCurrency, formatPercent, zeroBN } from 'utils/formatters/number';
 import { getExchangeRatesForCurrencies } from 'utils/currencies';
 import { Price } from 'queries/rates/types';
-import { getSynthDescription } from 'utils/futures';
+import { getSynthDescription, isEurForex } from 'utils/futures';
 import useMarketClosed from 'hooks/useMarketClosed';
+import { DEFAULT_FIAT_EURO_DECIMALS } from '../../../constants/defaults';
```

Comment:
Can simplify imports
```suggestion
import { DEFAULT_FIAT_EURO_DECIMALS } from 'constants/defaults';
```

Reply:
Oh, shame on me. I should disable auto-importing of vs code 🙃

Reply:
If there is a lint rule for it... that'd be sweet!